src/component/Dashboard/Propertypost.jsx

```diff
@@ -138,13 +138,13 @@ const Propertypost = () => {
                             </select>
                         </div>
                             <div>
-                                <label className='properpostLabel2'>Locality</label>
+                                <label className='propertypostLabel2'>Locality</label>
                                 <select name="mode" className="propertyText2" required='required'>
```

Comment:
I think I saw @mykelmatics feedback that you should make this a component so you can reuse it as many times as you want. Please, attend to that.